src/main/java/com/amazon/opendistroforelasticsearch/sql/utils/SQLFunctions.java

```diff
@@ -286,14 +286,17 @@ public static boolean isFunctionTranslatedToScript(String function) {
             case "log":
                 List<SQLExpr> logs = Lists.newArrayList();
                 for (int i = 0; i < paramers.size(); i++) {
-                    logs.add((SQLExpr) paramers.get(0).value);
+                    logs.add((SQLExpr) paramers.get(i).value);
                 }
```

Comment:
Any chance to remove this array list `logs`? I see more overhead than readability.

Reply:
It's done, thanks!